src/targets/services/stats.js

```diff
@@ -44,9 +46,15 @@ const client = new CozyClient({
   token: process.env.COZY_CREDENTIALS.trim()
 })
 
-BankTransaction.registerClient(client)
-
 const main = async () => {
+  Document.registerClient(client)
+
+  log('info', 'Fetching settings...')
+  let setting = await Settings.fetchWithDefault()
+
+  // The flag is needed to use local model when getting a transaction category ID
+  flag('local-model-override', setting.community.localModelOverride.enabled)
```

Comment:
Are we using a flag for a user preference here ? I think flags should not be used for that. Maybe we could have an issue about this ?

Reply:
The `getCategoryId` needs to know if the user enabled a setting (local model override) or not. This helper is used in multiple contexts and also in other helpers. So we would have to fetch and propagate the settings  in a lot of places. We decided to put this setting in a flag to avoid that. But yeah maybe we can find a better solution